.github/workflows/publish_connectors.yml

```diff
@@ -1,4 +1,4 @@
-name: Publish connectors on merge to master
+name: Connector Ops CI - Publish Connectors
```

Comment:
I like dropping the “on master”. If I’m looking at this action, I’m publishing a pre-release :)